eng/Versions.props

```diff
@@ -14,31 +15,21 @@
 
   <!-- Production Dependencies -->
   <PropertyGroup>
-    <MicrosoftAspNetCoreAllPackageVersion>2.2.0-preview2-35157</MicrosoftAspNetCoreAllPackageVersion>
+    <DotNetCoreSdkLKGVersion>3.0.100-preview-009722</DotNetCoreSdkLKGVersion>
```

Comment:
Is this duplicated with what's in global.json?  Should this read it from there instead?

Reply:
Yeah, we should take this from global.json. I will address this when working on enabling official builds based on arcade and dnceng, which I will start on next.